src/components/SearchResultsPage/SearchResultsPage.jsx

```diff
@@ -9,27 +9,34 @@ import dataArray from "../../data";
 
 const SearchResultsPage = () => {
   const [films, setFilms] = useState(dataArray);
-  const [isFilmActive, setIsFilmActive] = useState(false);

```

Comment:
isFilmActive is removed as activeFilm can be used instead, no need to duplicate it